docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  db_critick:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: user_critick
+      POSTGRES_PASSWORD: critick_db_test
+      POSTGRES_DB: db_critick
```

Comment:
can you verify if this part can be setup with values in env file please ? because we want the db info confidential even if this project is open-souce

Reply:
yes yes it is ok